packages/communication-react/review/communication-react.api.md

```diff
@@ -889,6 +889,8 @@ export interface Identifiers {
     participantList: string;
     // (undocumented)
     sendboxTextfield: string;
+    // (undocumented)
```

Comment:
Please document. All public API must be documented.

Reply:
Done